Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM node:10-alpine
+
+# Create app directory
+RUN mkdir -p /usr/src/express-sheets
+WORKDIR /usr/src/express-sheets
+
+# Install app dependencies
+COPY package*.json /usr/src/express-sheets/
+RUN npm install
+
+# Bundle app source
+COPY . /usr/src/express-sheets
+RUN npm run build
+
+EXPOSE 3000
+
+CMD npm run start
```

Comment:
I think we can use multistage docker build here so that we can reduce the image size.
@mderazon What do you think?

Reply:
I think we could

Reply:
@deepak-uttam 
@mderazon 
Please review now, I have created a multistage Dockerfile